source/api/commands/writefile.md

```diff
@@ -54,6 +53,12 @@ Pass in an options object to change the default behavior of `cy.writeFile()`.
 Option | Default | Description
 --- | --- | ---
 `log` | `true` | {% usage_options log %}
+`flag` | `w` | File system flag as used with {% url `fs.writeFile` https://nodejs.org/api/fs.html#fs_file_system_flags %}
```

Comment:
Can you display the options in alphabetical order please? Will add this as a note in our draft [Writing Style Guide](https://github.com/cypress-io/cypress-documentation/issues/853).

Reply:
hmm in this case only a couple of the flags are really relevant and feel like it might be information overload to include all of them. i'd be tempted to mention the main one (a+ for appending to a file instead of overwriting) in the description and then update it if we get feedback otherwise? also totally down to add them if we wanna stay consistent.